src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/FileIO/FileSystem.vb

```diff
@@ -14,14 +14,18 @@ Imports System.Security
 Imports System.Text
 
 Imports Microsoft.VisualBasic.CompilerServices
+#If PLATFORM_WINDOWS Then
+Imports Microsoft.VisualBasic.CompilerServices.NativeMethods
+Imports Microsoft.VisualBasic.CompilerServices.NativeTypes
+#End If
```

Comment:
The changes in this file are simply merging the contents of FileSystem*.vb so the differences with the reference source are clearer.